manifest.json

```
@@ -3,9 +3,9 @@
     "id": "zap",
     "packaging_format": 1,
     "description": {
-        "en": "Zap is a full featured social network application with identity/content mirroring across multiple servers (nomadic identity) but no federate with non-nomadic servers."
+        "en": "Zap is a full featured social network application with identity/content mirroring across multiple servers (nomadic identity) federating over ActivityPub and Zot6 protocols."
```

```suggestion
        "en": "A full featured social network app with identity/content mirroring across multiple servers federating over ActivityPub and Zot6 protocols."
```

That description is super-long ... here's an attempt to reduce it ..